css/lists.css

```diff
@@ -0,0 +1,26 @@
+.stacked-list .basic-card {
+	display: flex;
+	align-items: center;
+	gap: 1rem;
```

Comment:
Use global variables for spacing for properties like margin, padding, and gap to maintain consistency across the website

Reply:
Thanks will make that change after the assignment is complete

Reply:
Sure will make that change after finishing assignment